stories/setup/styles.scss

```diff
@@ -1,3 +1,41 @@
 body {
   padding: 24px;
 }
+
+.CodeShapeStory {
+  display: flex;
+}
+
+.CodeShapeStory-panel {
+  border: 1px solid;
+  padding: 10px;
+
+  &:first-child {
+    margin-right: 10px;
+  }
+}
+
+.CodeShapeStory-lineShape {
+  display: flex;
+  width: 100%;
+}
+
+.CodeShapeStory-actualCode,
+.CodeShapeStory-lineShape {
+  height: 1rem;
+  line-height: 1;
+  margin-top: 2px;
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+.CodeShapeStory-actualCode {
+  overflow: hidden;
+
+  pre {
```

Comment:
We tend to avoid referencing html tags in sass. Maybe we could introduce a className for that too? It could be `.CodeShapeStory-actualCode-line`.

Reply:
I think that was just for performance reasons, right? Since this is just storybook css, it shouldn't be a problem. 

I wasn't totally sure where this storybook css should go.